src/transformers/modeling_tf_bert.py

```diff
@@ -843,6 +847,80 @@ def call(self, inputs, **kwargs):
         return outputs  # prediction_scores, (hidden_states), (attentions)
 
 
+class TFBertLMHeadModel(TFBertForPreTraining, TFCausalLanguageModelingLoss):
```

Comment:
@LysandreJik @thomwolf split Bert into two as was done for PyTorch -> small break in backward compatibility here.